### PR TITLE
Transitions app

### PR DESCRIPTION
**Pull request by arvinth10**

### CODE REVIEW

File: transitions_app/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
```
Comment (arvinth10) [inline transitions_app/lib/main.dart:1]: File changed

File: transitions_app/lib/customPageTransitionsBuilder.dart [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
```
Comment (arvinth10) [inline transitions_app/lib/customPageTransitionsBuilder.dart]: New file

File: transitions_app/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# transitions_app
```
Comment (yenvanio) [inline transitions_app/README.md]: Update README.md to help readers run the app

File: transitions_app/lib/main.dart [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  @override
+  Widget build(BuildContext context) {
+    var rnd = Random();
```
Comment (yenvanio) [inline transitions_app/lib/main.dart]: Should this be a global variable?

**Review by yenvanio — APPROVED** [review]

**Review by yenvanio — APPROVED** [review]

**Pull request merged**